cdk/lib/__snapshots__/security-hq.test.ts.snap

```diff
@@ -151,6 +110,13 @@ Object {
             "PropagateAtLaunch": true,
             "Value": "guardian/security-hq",
           },
+          Object {
+            "Key": "LogKinesisStreamName",
```

Comment:
This'll allow us to use [devx-logs](https://github.com/guardian/devx-logs) (once we configure the AMIgo recipe). There's no impact otherwise.